tests/kernel/blktests.pm

```diff
@@ -27,18 +47,23 @@ sub run {
     my $quick   = get_required_var('BLK_QUICK');
     my $exclude = get_required_var('BLK_EXCLUDE');
     my $config  = get_required_var('BLK_CONFIG');
-    my $device  = get_required_var('BLK_DEVICE_ONLY');
+    my $devices = get_required_var('BLK_DEVICE_ONLY');
```

Comment:
Possible values of `BLK_DEVICE_ONLY` and other should be documented somewhere. Maybe in similar way like in https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/6d31c9f94f97a579d154b34c31793a7a6b4ba08b/tests/kernel/install_ltp.pm#L404 ?

Reply:
sure thing. I'm planning to do it. I just assume there will be still quite a lot of changes, so I would leave this for later, ok?

Reply:
ok